core/src/main/scala/cromwell/core/package.scala

```diff
@@ -16,7 +16,7 @@ package object core {
 
   type ErrorOr[+A] = ValidationNel[String, A]
   type LocallyQualifiedName = String
-  case class JobOutput(wdlValue: WdlValue, hash: Option[SymbolHash])
+  case class JobOutput(wdlValue: WdlValue, @deprecated hash: Option[SymbolHash] = None) // TODO: Remove this hash
```

Comment:
Fill in the `message` and `since` in your `@deprecated` tag:

```scala
$ sbt core/compile
[info] Loading global plugins from /Users/kshakir/.sbt/0.13/plugins
[info] Loading project definition from /Users/kshakir/src/cromwell/project
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[info] Set current project to cromwell (in build file:/Users/kshakir/src/cromwell/)
[info] Updating {file:/Users/kshakir/src/cromwell/}core...
[info] Resolving jline#jline;2.12.1 ...
[info] Done updating.
[info] Compiling 9 Scala sources to /Users/kshakir/src/cromwell/core/target/scala-2.11/classes...
[warn] /Users/kshakir/src/cromwell/core/src/main/scala/cromwell/core/package.scala:19: @deprecated now takes two arguments; see the scaladoc.
[warn]   case class JobOutput(wdlValue: WdlValue, @deprecated hash: Option[SymbolHash] = None) // TODO: Remove this hash
[warn]                                             ^
[warn] /Users/kshakir/src/cromwell/core/src/main/scala/cromwell/core/package.scala:19: value hash in class JobOutput is deprecated
[warn]   case class JobOutput(wdlValue: WdlValue, @deprecated hash: Option[SymbolHash] = None) // TODO: Remove this hash
[warn]                                                        ^
[warn] /Users/kshakir/src/cromwell/core/src/main/scala/cromwell/core/package.scala:19: value hash in class JobOutput is deprecated
[warn]   case class JobOutput(wdlValue: WdlValue, @deprecated hash: Option[SymbolHash] = None) // TODO: Remove this hash
[warn]              ^
[warn] three warnings found
[success] Total time: 16 s, completed Jun 1, 2016 8:22:17 PM
$ 
```